server/builtin/field_app_id.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package builtin
+
+import (
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+)
+
+const (
+	LookupAny              = ""
+	LookupDisabledApps     = "disabled"
+	LookupEnabledApps      = "enabled"
+	LookupInstalledApps    = "installed"
+	LookupNotInstalledApps = "not_installed"
+)
+
+func (a *builtinApp) appIDField(lookupType string, autocompletePos int, isRequired bool, loc *i18n.Localizer) apps.Field {
+	lookupCall := newUserCall(pLookupAppID)
+	lookupCall.State = lookupType
+
+	return apps.Field{
+		Name: fAppID,
+		Type: apps.FieldTypeDynamicSelect,
+		Description: a.conf.I18N().LocalizeDefaultMessage(loc, &i18n.Message{
+			ID:    "field.appID.description",
+			Other: "Select an App or enter the App ID",
+		}),
+		Label: a.conf.I18N().LocalizeDefaultMessage(loc, &i18n.Message{
+			ID:    "field.appID.label",
+			Other: "app",
+		}),
+		AutocompletePosition: autocompletePos,
+		IsRequired:           isRequired,
+		SelectDynamicLookup:         lookupCall,
```

Comment:
`WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive.`

@hanzei you know anything about this?

Reply:
The go-to version for `golangci-lint` is v1.33.0 right now. That one should complain about the issue above

Reply:
I installed 1.33 and it does not appear to be catching `gofmt` issues either

Reply:
Yep, can confirm that. I don't know why this happens. Seems to be a bug in `golangci-lint`.